pkg/policy/defaultRules_test.go

```diff
@@ -0,0 +1,137 @@
+package policy
+
+import (
+	_ "embed"
+	"encoding/json"
+	"fmt"
+	"reflect"
+	"testing"
+
+	"github.com/datreeio/datree/pkg/fileReader"
+	"github.com/datreeio/datree/pkg/jsonSchemaValidator"
+	"github.com/ghodss/yaml"
+	"github.com/stretchr/testify/assert"
+)
+
+//go:embed defaultRulesSchema.json
+var defaultRulesSchemaContent string
+
+const defaultRulesYamlPath = "./defaultRules.yaml"
+
+func TestDefaultRulesFileExists(t *testing.T) {
+	_, fileReadError := getFileFromPath(defaultRulesYamlPath)
+
+	assert.Nil(t, fileReadError)
+}
+
+func TestDefaultRulesFileFitsJSONSchema(t *testing.T) {
+	validationError := validateYamlUsingJSONSchema(defaultRulesYamlPath, defaultRulesSchemaContent)
+
+	assert.Nil(t, validationError)
+}
+
+func TestDefaultRulesHasUniqueNamesInRules(t *testing.T) {
+	defaultRulesFileContent, _ := getFileFromPath(defaultRulesYamlPath)
```

Comment:
I am not sure what is the best solution but I think that we read the same file 3 / more times in this test, maybe we can read it once and use it's content in all Test functions